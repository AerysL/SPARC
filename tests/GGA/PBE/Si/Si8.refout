***************************************************************************
*                           SPARC (May 11, 2020)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun May 17 23:26:29 2020                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.2 10.2 10.2 
LATVEC:
1 0 0 
-0.292371704722737 0.956304755963035 0 
0.17364817766693 -0.110492654830881 0.978589640054184 
FD_GRID: 25 25 25
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.0009999935878
CHEB_DEGREE: 20
NSTATES: 30
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: PBE/Si/Si8
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 24
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 2 3 4
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  24
Mesh spacing                       :  0.408 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  PBE/Si/Si8.out_1
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  PBE/Si/Si.psp8
Pseudocharge radii of atom type 1  :  7.75 7.75 7.75 (x, y, z dir)
Number of atoms of type 1          :  8
Estimated total memory usage       :  24.80 MB
Estimated memory per processor     :  1.03 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9139148584E+00        2.310E-02        0.221
2            -3.9153182030E+00        1.334E-02        0.057
3            -3.9155076832E+00        6.956E-03        0.055
4            -3.9155504729E+00        2.783E-03        0.054
5            -3.9155545559E+00        1.198E-03        0.055
6            -3.9155552437E+00        5.768E-04        0.053
7            -3.9155554497E+00        3.088E-04        0.052
8            -3.9155555011E+00        1.201E-04        0.052
9            -3.9155555499E+00        4.219E-05        0.050
10           -3.9155555272E+00        2.151E-05        0.048
11           -3.9155555284E+00        7.648E-06        0.057
12           -3.9155555543E+00        3.830E-06        0.045
13           -3.9155555230E+00        1.468E-06        0.047
14           -3.9155555101E+00        5.121E-07        0.044
Total number of SCF: 14    

k = [0.000, 0.000, 0.000]
Occupation of state 27 = 0.000000000000000.
Occupation of state 30 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9155555101E+00 (Ha/atom)
Total free energy                  : -3.1324444081E+01 (Ha)
Band structure energy              :  1.3723951814E+00 (Ha)
Exchange correlation energy        : -9.7764028624E+00 (Ha)
Self and correction energy         : -1.0219468888E+02 (Ha)
Entropy*kb*T                       : -2.0844149154E-04 (Ha)
Fermi level                        :  2.3355824585E-01 (Ha)
RMS force                          :  1.3572950174E-01 (Ha/Bohr)
Maximum force                      :  1.3577064940E-01 (Ha/Bohr)
Time for force calculation         :  0.009 (sec)
Pressure                           :  1.7874718812E+01 (GPa)
Maximum stress                     :  2.2523298109E+01 (GPa)
Time for stress calculation        :  0.058 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  1.036 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
