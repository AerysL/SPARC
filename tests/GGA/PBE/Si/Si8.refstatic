Total free energy (Ha): -3.132444408081752E+01
Atomic forces (Ha/Bohr):
 -5.4795959085E-02   4.8530299125E-02  -1.1424518109E-01
 -5.4840180116E-02   4.8557959076E-02  -1.1431687755E-01
 -5.4818359976E-02   4.8537718473E-02  -1.1428963781E-01
 -5.4815650462E-02   4.8545730054E-02  -1.1428931248E-01
  5.4813912221E-02  -4.8563583707E-02   1.1423988848E-01
  5.4835433346E-02  -4.8531901667E-02   1.1423821761E-01
  5.4809152409E-02  -4.8535505700E-02   1.1433298430E-01
  5.4811651663E-02  -4.8540715655E-02   1.1432991857E-01
Stress (GPa): 
 -1.6458966416E+01  -8.3625562463E+00  -3.8854036467E+00 
 -8.3625562463E+00  -2.2523298109E+01   7.0462877054E+00 
 -3.8854036467E+00   7.0462877054E+00  -1.4641891910E+01
