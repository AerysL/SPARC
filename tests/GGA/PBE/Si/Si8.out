***************************************************************************
*                           SPARC (Mar 13, 2019)                          *
*   Copyright (c) 2018 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                 Start time: Sun Mar 17 18:05:23 2019                    *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.200000 10.200000 10.200000 
LATVEC:
1.000000 0.000000 0.000000 
-0.292372 0.956305 0.000000 
0.173648 -0.110493 0.978590 
FD_GRID: 25 25 25
FD_ORDER: 12
BOUNDARY_CONDITION: 2
KPOINT_GRID: 1 1 1
ELEC_TEMP: 315.775131
CHEB_DEGREE: 20
NSTATES: 30
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGER: 3
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Si8
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 25
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 2 3 4
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  25
Mesh spacing                       :  0.408000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ./Si8.out
Forces printed to                  :  ./Si8.force
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  ./Si.psp8
Pseudocharge radii of atom type 1  :  7.34 7.34 7.34 (x, y, z dir)
Number of atoms of type 1          :  8
====================================================================
                    Self Consistent Field (SCF#0)                     
====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.0236238703E+00        4.817E-02        0.840
2            -3.9157679398E+00        2.849E-02        0.321
3            -3.9168487385E+00        1.279E-02        0.313
4            -3.9173496700E+00        8.958E-03        0.308
5            -3.9174927484E+00        3.708E-03        0.306
6            -3.9169268613E+00        1.898E-03        0.300
7            -3.9161480126E+00        1.180E-03        0.297
8            -3.9153743520E+00        3.806E-04        0.294
9            -3.9152690803E+00        1.825E-04        0.285
10           -3.9154916467E+00        6.508E-05        0.281
11           -3.9155212152E+00        3.175E-05        0.268
12           -3.9155658878E+00        1.288E-05        0.270
13           -3.9155622552E+00        6.020E-06        0.257
14           -3.9155601411E+00        2.932E-06        0.255
15           -3.9155574589E+00        1.533E-06        0.248
16           -3.9155557532E+00        6.537E-07        0.244
====================================================================
                                Energy                     
====================================================================
Total free energy                  :  -3.1324446026E+01 (Ha)
Band structure energy              :   1.3723951098E+00 (Ha)
Exchange correlation energy        :  -9.7764010547E+00 (Ha)
Self and correction energy         :  -1.0219468890E+02 (Ha)
Entropy*kb*T                       :  -2.0844605215E-04 (Ha)
Fermi level                        :   2.3355827524E-01 (Ha)
Atomic forces (Ha/Bohr):
     -0.0547963663       0.0485301658      -0.1142444235
     -0.0548397870       0.0485584418      -0.1143164394
     -0.0548185108       0.0485381394      -0.1142900820
     -0.0548155510       0.0485456094      -0.1142903141
      0.0548135214      -0.0485639285       0.1142393648
      0.0548356395      -0.0485317586       0.1142374676
      0.0548088966      -0.0485360352       0.1143335948
      0.0548121577      -0.0485406340       0.1143308318
Average force                      :  0.135729619130968 (Ha/Bohr)
Maximum force                      :  0.135770294353994 (Ha/Bohr)
Time for force calculation         :  0.047 (sec)
Stress (GPa) : 
 -16.459104985185167 -8.362616055688116 -3.885250606342598 
 -8.362616055688116 -22.523282662261412  7.046273675642449 
 -3.885250606342598  7.046273675642449 -14.641765120016498 
Pressure                           :  17.874717589154358 (GPa)
Maximum stress                     :  22.523282662261412 (GPa)
Time for stress calculation        :   0.233 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  5.618 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*                    Collaborator: John E. Pask (LLNL)                    *
*        Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500)    *
***************************************************************************
                                                                           
